rc/windowing/sway.kak

```diff
@@ -0,0 +1,51 @@
+provide-module sway %{
+
+# Ensure we're actually in Sway
+evaluate-commands %sh{
+    [ -z "${kak_opt_windowing_modules}" ] || 
+    [ -n "$SWAYSOCK" ] ||
+    echo 'fail SWAYSOCK is not set'
+}
+
+require-module 'wayland'
+
+define-command sway-focus-pid -hidden %{
+    evaluate-commands %sh{
+        pid=$kak_client_pid
+
+        # Try to focus a window with the current PID, walking up the tree of 
+        # parent processes until the focus eventually succeeds
+        while ! swaymsg [pid=$pid] focus > /dev/null ; do
```

Comment:
If `swaymsg` sends an error to stderr it will appear in kakoune's `*debug*` buffer. So as you traverse the hierarchy looking for the correct pid you might see a lot of these messages generated by `swaymsg` which are not really errors but rather us hunting for the correct pid. So, should we have something like:

```
 while ! swaymsg [pid=$pid] focus > /dev/null 2> /dev/null; do
...
```

here instead?